README.md

```diff
@@ -78,3 +78,51 @@ be heard and the call hung up.
 To uninstall all files associated with the example issue the following command:
 
     make uninstall-example
+
+## Asterisk Channel Variables
+
+You can use the following Respoke session information to manage them inside the asterisk channel:
+
+- respoke\_session\_local
```

Comment:
We have modified the variable names according to the asterisk convention (i.e. RESPOKE_SESSION_ID), thus UPPERCASE. We have still kept the _, like many other asterisk variables (i.e. ${SIP_CODEC}, ${TOUCH_MONITOR}, ${TRANSFER_CONTEXT} and many others).
The function option is currently not easy to be done. 

Thank you for your suggestions.